kubeflow/sync-repos.sh

```diff
@@ -8,6 +8,16 @@ if [ -n "${GENIENLP_VERSION}" ] && [ "${GENIENLP_VERSION}" != "${GENIENLP_HEAD}"
   pip3 install --upgrade --use-feature=2020-resolver -e .
 fi
 
+if [ -d /opt/bootleg/ ] ; then
+  cd /opt/bootleg/
+  BOOTLEG_HEAD=`git rev-parse HEAD`
+  if [ -n "${BOOTLEG_VERSION}" ] && [ "${BOOTLEG_VERSION}" != "${BOOTLEG_HEAD}" ]; then
+    git fetch
+    git checkout ${BOOTLEG_VERSION}
+    pip3 install --upgrade --use-feature=2020-resolver -e .
```

Comment:
This pip install has no effect, because bootleg does not use setup.py to specify dependencies. You need to install their requirements.txt